### PR TITLE
fixed file extension

### PR DESCRIPTION
**Pull request by xiaobfly**
google::SetLogFilenameExtension(".log");

before:
Test.log20210401-161508.34056

after:
Test20210401-161508.34056.log

### CODE REVIEW

**Comment by smallb** [comment]
google::SetLogFilenameExtension(".log");

before:
Tset.log20210401-161508.34056

after:
Tset20210401-161508.34056.log

**Comment by smallb** [comment]
> @googlebot I signed it!

**Comment by xiaobfly** [comment]
> @googlebot I fix

File: src/utilities.cc [hunk 1]
```diff
@@ -60,7 +60,7 @@ using std::string;
 
 _START_GOOGLE_NAMESPACE_
 
-static const char* g_program_invocation_short_name = NULL;
+static std::string g_program_invocation_short_name;
```
Comment (sergiud) [inline src/utilities.cc:63]: How are these changes related to the file extension problem?

**Review by sergiud — CHANGES_REQUESTED** [review]

File: src/utilities.cc (hunk 1 above)
Comment (xiaobfly) [reply src/utilities.cc:63]: > How are these changes related to the file extension problem?

this commit: 4fca1b96730814666a1823f595dba5912a272d43

Comment (sergiud) [reply src/utilities.cc:63]: Sorry, I do not understand. You need to explain why are you changing those lines.

Comment (xiaobfly) [reply src/utilities.cc:63]: g_program_invocation_short_name is a const char*, if call InitGoogleLoggingUtilities  not in main or arg0 is a local value, g_program_invocation_short_name maybe has lifetime problem.

Comment (sergiud) [reply src/utilities.cc:63]: Please create a separate PR for this issue.

Comment (xiaobfly) [reply src/utilities.cc]: new issue: https://github.com/google/glog/issues/633#issue-848901031
new PR: https://github.com/google/glog/pull/634#issue-607763810

**Review by sergiud — APPROVED** [review]

**Comment by aesophor** [comment]
If the user has set an `filename_extension` for the log files, then the log cleaner will be broken because the filename format has been changed.

https://github.com/google/glog/blob/e51790bf33d7d1f0e6b7e703a29a972b570293a4/src/logging.cc#L1375-L1432

**Comment by xiaobfly** [comment]
> IsLogFromCurrentProject

I will fix it.
